helm/high-availability.yaml

```diff
@@ -0,0 +1,10 @@
+# High Availability (HA) configuration for OLMv1
```

Comment:
I think this file should be moved to `helm/olmv1` and name it `values-high-availability.yaml` so it is clear that it contains chart values.

Reply:
We've been placing values files directly into the `helm` directory, so I think the locations is fine for now.
We can look at moving values files "into the charts" in another PR.